akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -96,7 +96,7 @@ private[akka] class Shard(
   extractShardId: ShardRegion.ExtractShardId,
   handOffStopMessage: Any) extends Actor with ActorLogging {
 
-  import ShardRegion.{ handOffStopperProps, EntityId, Msg, Passivate }
+  import ShardRegion.{ handOffStopperProps, EntityId, Msg, Passivate, ShardInitialized }
```

Comment:
Sorry, I forgot answer to you here.
Sometimes persistent shard could be failure during a replay process due to temporary journal unavailability. The user messages which sent to such shard just after creation, but before a failure would be lost. This code removes that problem.